app/ledger.js

```diff
@@ -190,6 +191,21 @@ if (ipc) {
 
     event.returnValue = { context: ctx, rules: publisherInfo._internal.ruleset.cooked }
   })
+
+  ipc.on(messages.CHANGE_SETTING, (event, key, value) => {
+    if (!client) return
+
+    if (key === settings.PAYMENTS_MONTHLY_AMOUNT) {
+      var amount = parseInt(value, 10)
+      var bravery = client.getBraveryProperties()
+
+      if (isNaN(amount) || (amount <= 0)) return
+
+      underscore.extend(bravery.fee, { amount: amount })
```

Comment:
i just changed it so now this code is correct as long as `amount` corresponds to the currency in ledger-state